doc/rm/comportability_specification/index.md

```diff
@@ -440,18 +440,70 @@ Unlike interrupts, there is no software component to alerts at the peripheral, t
 
+Alerts of comportable IPs in the system must be in either of the following two categories:
+
+1. *Recoverable*, one-time triggered alerts.
+This category is for regular alerts that are due to recoverable error conditions.
+The alert sender transmits one single alert event when the corresponding error condition is asserted.
+
+2. *Fatal* alerts that are continuously triggered until reset.
+This category is for highly critical alerts that are due to terminal error conditions.
+The alert sender continuously transmits alert events until the system is reset.
+
+It is recommended that fatal alerts also trigger local security countermeasures, if they exist.
+For example, a redundantly encoded FSM that is glitched into an invalid state is typically considered to be a fatal error condition.
+In this case, a local countermeasure could be to move the FSM into a terminal error state in order to render the FSM inoperable until the next reset.
+
+The table below lists a few common error conditions and the recommended alert type for each of those errors.
+
+Error Event                                                | Regular IRQ | Recoverable Alert | Fatal Alert
+-----------------------------------------------------------|-------------|-------------------|-------------
+ECC correctable in NVM (OTP, Flash)                        | x           | x                 |
+ECC uncorrectable in Flash                                 | x           | x                 |
+ECC uncorrectable in OTP                                   | (x)         |                   | x
+Any ECC / Parity error in SRAMs or register files          | (x)         |                   | x
+Glitch detectors (e.g., invalid FSM encoding)              | (x)         |                   | x
+Incorrect usage of security IP (e.g., AES config register) | x           | x                 |
+Incorrect usage of regular IP                              | x           |                   |
+
+(x): optional
+
+The column "Regular IRQ" indicates whether the corresponding error condition should also send out a regular IRQ.
+Fatal alerts are not required to send out regular IRQs, since they cannot be recovered via SW actions.
+
+### Defining Alerts
+
+The Hjson configuration file defined above specifies all that needs to be known about the alerts in the standard case.
+The following sections specify what comes out of various tools based upon the simple list defined in the above example.
+
+In terms of naming convention, recoverable alerts must be prefixed with `recov_*`, whereas fatal alerts must be prefixed with `fatal_*`.
+If a module only outputs one alert of each type, it is permissible to name the alerts `recoverable`/`recov` and/or `fatal`.
+
+### Test Alert Register Creation
+
+For every peripheral, by default, one register named `ALERT_TEST` is **automatically** created.
+This can be overridden within the `reggen` tool by specifying `no_auto_alert_regs = true`.
```

Comment:
Providing the ability to disable this is probably not a good idea, given that we plan to build an automated test with it.

Reply:
Yeah I agree. I think this feature has been added for two reasons:
- disable generation of the regs in case they need to be customized
- disable generation of the regs if no IRQs/alert are defined

The first point above is only intended for "super users" and special cases.
The second point is obsolete, since in the meantime, reggen can automatically infer whether these autoregs are needed or not, based on whether the `interrupt_list` or `alert_list` keys are present in the Hjson or not.

So I think it would be best to not advertise that this feature exists.
I'll delete that sentence in the interrupt section also, ok?

Reply:
Yes, not advertising this feature here sgtm.

Reply:
Ok good. I removed these lines in both the interrupt and alert section.